examples/maxq10xx-wolfssl-client.c

```diff
@@ -0,0 +1,335 @@
+/* maxq10xx-wolfssl-client.c
+ *
+ * Based on client-tls-pkcallback.c
+ *
+ * Copyright (C) 2006-2022 wolfSSL Inc.
+ *
+ * This file is part of wolfSSL.
+ *
+ * wolfSSL is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * wolfSSL is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA
+ */
+
+/* This example shows how to write a simple TLS client that uses the features
+ * features of the Analog Devices MAXQ 1065 and 1080. */
+
+#include <stdlib.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/socket.h>
+#include <arpa/inet.h>
+#include <netinet/in.h>
+#include <unistd.h>
+
+/* wolfSSL */
+#ifndef WOLFSSL_USER_SETTINGS
+    #include <wolfssl/options.h>
+#endif
+#include <wolfssl/ssl.h>
+#include <wolfssl/wolfcrypt/sha256.h>
+#include <wolfssl/wolfcrypt/cryptocb.h>
+#include <wolfssl/wolfcrypt/ecc.h>
+#include <wolfssl/wolfcrypt/rsa.h>
+#include <wolfssl/wolfcrypt/asn.h>
+#include <wolfssl/wolfcrypt/error-crypt.h>
+
+#if (defined(WOLFSSL_MAXQ1065) || defined(WOLFSSL_MAXQ108x)) && \
+    defined(HAVE_PK_CALLBACKS) && defined(WOLF_CRYPTO_CB) && \
+    !defined(NO_PSK) && !defined(HAVE_EXTENDED_MASTER) && \
+    defined(NO_WOLFSSL_SERVER)
+
+/* -------------- */
+/* Configurations */
+/* -------------- */
+
+/* Please define this if you want wolfSSL's debug output */
```

Comment:
Just personal preference, but I don't think we need to say please for all our comments. :)

Reply:
It signals that the reader has something they need to do.